src/ffmpeg-util.jl

```diff
@@ -63,15 +63,12 @@ struct VideoStreamOptions
             (compression === nothing) && (compression = 20)
         end
 
-        if format == "gif"
-            (loop === nothing) && (loop = 0)
-        end
+        (loop === nothing) && (loop = 0)
 
         # items are name, value, allowed_formats
         allowed_kwargs = [("compression", compression, ("mp4", "webm")),
                           ("profile", profile, ("mp4",)),
-                          ("pixel_format", pixel_format, ("mp4",)),
-                          ("loop", loop, ("gif",))]
```

Comment:
I think that `loop` is given to the rendered GIF here, not the HTML.  Can we add this back in?

Reply:
Makes sense - thanks!